src/test/data/script_tests.json

```diff
@@ -2527,6 +2527,217 @@
 ["0x050000000001", "CHECKSEQUENCEVERIFY", "CHECKSEQUENCEVERIFY", "UNSATISFIED_LOCKTIME",
   "CSV fails if stack top bit 1 << 31 is not set, and tx version < 2"],
 
+["OP_CAT tests"],
+[
+    [
+        "aa",
+        "bb",
+        "7e4c02aabb87",
+        "c0d6889cb081036e0faefa3a35157ad71086b123b2b144b649798b494c300a961d",
+        0.00000001
+    ],
+    "",
+    "0x51 0x20 0x15048ed3a65748549c27b671936987093cf73a4c9cb18522a74fb9553060ca99",
+    "P2SH,WITNESS,TAPROOT",
```

Comment:
Yeah, I see that now. Something is strange. It makes sense that the first couple tests that check for valid use of CAT are passing. But Shouldn't the failure testcases still fail for b/c they being executed with OP_SUCCESS default behavior?

Reply:
They were passing for me because I was using an older version of the code that automatically activated OP_CAT. I just pulled the latest from this branch and now they are broken exactly as they should be